src/tiles/grid/tile-stats.ts

```diff
@@ -0,0 +1,104 @@
+import { css, CSSResultGroup, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { favoriteFilledIcon } from '../../assets/img/icons/favorite-filled';
+import { reviewsIcon } from '../../assets/img/icons/reviews';
+import { uploadIcon } from '../../assets/img/icons/upload';
+import { viewsIcon } from '../../assets/img/icons/views';
+
+import { formatCount } from '../../utils/format-count';
+
+@customElement('tile-stats')
+export class TileStats extends LitElement {
+  @property({ type: String }) mediatype?: string;
+
+  @property({ type: Number }) itemCount?: number;
+
+  @property({ type: Number }) viewCount?: number;
+
+  @property({ type: Number }) favCount?: number;
+
+  @property({ type: Number }) commentCount?: number;
+
+  render() {
+    console.log('tile-stats here');
+    return html`
+      <div class="stats-wrapper">
+        <div id="stats-holder">
+          <div class="col">
+            <mediatype-icon
+              .mediatype=${this.mediatype}
+              style="--iconHeight:25px; --iconWidth:25px;"
+            >
+            </mediatype-icon>
+          </div>
+          <div class="col">
+            ${this.mediatype === 'account' ? uploadIcon : viewsIcon}
+            <p class="status-text">
+              ${formatCount(
+                this.mediatype === 'account' ? this.itemCount : this.viewCount,
+                'short',
+                'short'
+              )}
+            </p>
+          </div>
+          <div class="col">
+            ${favoriteFilledIcon}
+            <p class="status-text">
+              ${formatCount(this.favCount, 'short', 'short')}
+            </p>
+          </div>
+          <div class="col">
+            ${reviewsIcon}
+            <p class="status-text">
+              ${formatCount(this.commentCount, 'short', 'short')}
+            </p>
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  static get styles(): CSSResultGroup {
+    return css`
+      .stats-wrapper {
+        height: 35px;
```

Comment:
Open question to everyone, are we now supporting using px unit instead of rem/em units? At PB side, we totally against it unless its really needed. @iisa @mc2 @dualcnhq

Reply:
`px` was set in the mockups so i just followed it.

cc @pezvi